pom.xml

```diff
@@ -60,6 +60,9 @@
     <changelist>999999-SNAPSHOT</changelist>
     <scmTag>HEAD</scmTag>
     <spotbugs.excludeFilterFile>${project.basedir}/../src/spotbugs/spotbugs-excludes.xml</spotbugs.excludeFilterFile>
+
+    <!-- Generate metadata for reflection on method parameters -->
+    <maven.compiler.parameters>true</maven.compiler.parameters>
```

Comment:
Better style to use plugin `configuration`.

Reply:
> Better style

Do you have a reference to a style guide?

Reply:
Could not find one. I wrote up what I know: https://stackoverflow.com/a/67794133/12916

Reply:
OK, this still seems like a fairly subjective argument to me, but I trust your judgment.